hbjs.js

```diff
@@ -533,7 +533,23 @@ function hbjs(Module) {
     return trace;
   }
 
-  function get_version() {
+  function version() {
+    var major = exports.malloc(4);
+    var minor = exports.malloc(4);
+    var patch = exports.malloc(4);
+    exports.hb_version(major, minor, patch);
+    var version = [
+      heapu32[major / 4],
+      heapu32[minor / 4],
+      heapu32[patch / 4]
+    ];
```

Comment:
would an object be better?
```suggestion
    var version = {
      major: heapu32[major / 4],
      minor: heapu32[minor / 4],
      patch: heapu32[patch / 4]
    };
```